builtin/providers/aws/resource_aws_dynamodb_table.go

```diff
@@ -158,6 +159,20 @@ func resourceAwsDynamoDbTable() *schema.Resource {
 					return hashcode.String(buf.String())
 				},
 			},
+			"stream_enabled": &schema.Schema{
+				Type: schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			"stream_view_type": &schema.Schema{
+				Type: schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
@phinze ok, this is currently building in travis and then it's ready for you again :)